controller/destination/profile_listener.go

```diff
@@ -0,0 +1,352 @@
+package destination
+
+import (
+	"errors"
+	"net/http"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	pb "github.com/linkerd/linkerd2-proxy-api/go/destination"
+	pbHttp "github.com/linkerd/linkerd2-proxy-api/go/http_types"
```

Comment:
Our convention when renaming protobuf imports is to use "Pb" as a suffix, instead of "pb" as a prefix. So this should be "httpPb".